libs/state/docs/concepts-and-best-practices.md

```diff
@@ -0,0 +1,133 @@
+# General
+
+**Component Shell and Folder**
+
+- `@Input` bindings are setter
```

Comment:
at least explain the different ways of using inputs (single values vs. observable configuration input) and state pros of the observable input (less change detections -> performance increase)

Reply:
> single values vs. observable configuration input
I would put this in a separate piece. the snippets would be a good place.

This could be too much at one point I believe.

Reply:
Just checked it. We have it under snippets.